build.nix

```diff
@@ -335,24 +345,21 @@ let
       runHook postConfigure
     '';
 
-    buildPhase =
```

Comment:
This was not changed, only reformatted (one nesting level less, as per rest of the scripts in this file).